qa/workunits/rados/test_rados_tool.sh

```diff
@@ -52,18 +52,30 @@ run() {
 DNAME="`dirname $0`"
 DNAME="`readlink -f $DNAME`"
 RADOS_TOOL="`readlink -f \"$DNAME/../rados\"`"
-if ! test -f $RADOS_TOOL ; then
-    RADOS_TOOL=$(which rados)
+if [ ! -f "$RADOS_TOOL" ] ; then
```

Comment:
the reason why we do `$(which rados)` is for the qa run, as the path name ("qa/workunits/rados/test_rados_tool.sh") suggests, `test_rados_tool.sh` is a workunit used by ceph-qa-suite to build the workload for the cluster under testing. in that case, "rados" cli is installed by the package manager (rpm or dpkg), so, `which rados` is a reliable way to locate "rados" in that case.